frontend/spec/features/products_sorting_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
```

Comment:
@Vegann let's also add a spec to API (and update the docs) and also to the finder class itself :) thank you!